src/main/resources/assets/js/application/interviewer/services/tdprPersonService.js

```diff
@@ -0,0 +1,21 @@
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function(angular, tdprInterviewerModule) {
+    tdprInterviewerModule.service("tdprPersonService", function($http) {
+
+        var person;
+
+        this.getPersonDetails = function(id) {
+            return $http.get("/api/person/" + id).then(function(response){
+                person = response.data;
+                return response;
+            });
+        };
+
+        this.updatePersonDetails = function(person){
+            return $http.put("/api/person/create", person);
```

Comment:
a PUT to an endpoint called create? This is peculiar and not compliant with REST rules. a put should be to /api/person/{id} if you want to update